studyzen/Set/FlashCardSetController.cs

```diff
@@ -0,0 +1,97 @@
+using Microsoft.AspNetCore.Mvc;
+using StudyZen.Common;
+using StudyZen.FlashCards.Requests;
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+
+
+namespace StudyZen.FlashCardSets;
+
+[ApiController]
+[Route("[controller]")]
+public sealed class FlashCardSetController : ControllerBase
+{
+    private readonly IFlashCardSetService _flashCardSetService;
+
+    public FlashCardSetController(IFlashCardSetService flashCardSetService)
+    {
+        _flashCardSetService = flashCardSetService;
+    }
+
+    
+    [HttpPost("add-flashcardset")]
+        public IActionResult AddFlashCardSet([FromBody] CreateFlashCardSetRequest? request)
+        {
+            request = request.ThrowIfRequestArgumentNull(nameof(request));
+
+            var flashCardSetId = _flashCardSetService.AddFlashCardSet(request);
+
+            var response = new
+            {
+                FlashCardSetId = flashCardSetId,
+                SetName = request.SetName,
+                Color = request.Color,
+                LectureId = request.LectureId
+            };
+
+            return CreatedAtAction(nameof(GetFlashCardSet), new { flashCardSetId = flashCardSetId }, response);
+        }
+
+    [HttpGet("get-flashcardset/{flashCardSetId}")]
+    public IActionResult GetFlashCardSet(int flashCardSetId)
+    {
+        
+        var flashCardSet = _flashCardSetService.GetFlashCardSet(flashCardSetId);
+       
+        if (flashCardSet == null)
```

Comment:
change if/else to ternary operator         `return flashCardSet== null ? NotFound() : Ok(flashCardSet);`